pyvrp/solve.py

```diff
@@ -64,7 +64,7 @@ def __init__(
         perturbation_ops: list[
             type[PerturbationOperator]
         ] = PERTURBATION_OPERATORS,
-        num_perturbations: int = 10,
+        num_perturbations: int = 25,
```

Comment:
Should this be part of some `PerturbationParams` object, together with the perturb ops? It doesn't feel as independent from other parameters as, say, the `display_interval` argument did.

Reply:
Getting the perturbation operators into PerturbationParams felt a bit tricky. Some things that came up during my attempt:
- What's the default for perturbation operators? I'd say all perturbation operators, but this results in LS having by default perturbation operators while having no node operators.
- When do we add perturbation operators if they are part of the params? If we add at construction then we get the above issue.

I think it's better to keep the perturbation_ops like the other operators, but `PerturbationParams` is ok to have to store the `num_perturbations` parameter.